web_src/js/utils/dom.js

```diff
@@ -51,8 +51,22 @@ export function isElemHidden(el) {
   return res[0];
 }
 
-export function queryElemSiblings(el, selector = '*') {
-  return Array.from(el.parentNode.children).filter((child) => child !== el && child.matches(selector));
+function applyElemsCallback(elems, fn) {
+  if (fn) {
+    for (const el of elems) {
+      fn(el);
+    }
+  }
+  return elems;
+}
+
+export function queryElemSiblings(el, selector = '*', fn = null) {
```

Comment:
```suggestion
export function queryElemSiblings(el, selector = '*', fn) {
```

Reply:
No reason to default to null when default `undefined` works just as well.

Reply:
1. In most languages (maybe all except JS), `foo(a, b=1, c)` is incorrect syntax, and it looks really ugly.
2. `null` is shorter than `undefined`, and in this case it just works well.

Feel free to change to what you prefer.

Reply:
`null` is problematic with default parameters because they only work with undefined.

More reasons to avoid null in https://github.com/sindresorhus/meta/discussions/7, I don't agree to remove null completely in code as I still like to use it in certain cases.